util/embed_builder.py

```diff
@@ -27,9 +27,15 @@ def __init__(
     def build(self) -> discord.Embed:
         embed = discord.Embed(
             title=self.title[:256],
-            description=self.description[:4096],
+            description=self.description[:1024],
             color=self.color,
         )
+
+        # Ensure the description stops at the end of a word
+        last_space_index = embed.description.rfind(" ")
+        if last_space_index != -1:
+            embed.description = embed.description[:last_space_index]
```

Comment:
This will pretty much always cut off the last word since we don't put spaces at the end of our sentences.